src/utils/backend/classroom/classroom.ts

```diff
@@ -89,18 +131,70 @@ export async function updateClassroom(
   return { data: updatedClass, error: null };
 }
 
+export async function addAdvisorToClassroom(
+  teacherID: number,
+  classID: number
+): Promise<{ data: ClassroomTable | null; error: PostgrestError | null }> {
+  const { data: classroom, error: classroomSelectionError } = await supabase
+    .from<{ advisors: number[]; number: number; year: number }>("classroom")
+    .select("advisors, number, year")
+    .match({ id: classID, year: getCurrentAcedemicYear() })
+    .limit(1)
+    .single();
+
+  if (!classroom || classroomSelectionError) {
+    return { data: null, error: classroomSelectionError };
+  }
+
+  const { data: updatedClassroom, error: classroomUpdatingError } =
+    await supabase
+      .from<ClassroomTable>("classroom")
+      .update({ advisors: [...classroom.advisors, teacherID] })
+      .eq("id", classID)
+      .single();
+  // UPDATE contacts value ... WHERE id = ?;
```

Comment:
What is this?

Reply:
just a SQL equivalent so we could remove it if you want